.dockerignore

```diff
@@ -0,0 +1,4 @@
+bin
```

Comment:
Could this include `.git` also?

Thanks for adding this- be useful to exclude as much as we can.